app/pages/about.cjsx

```diff
@@ -104,10 +102,13 @@ module.exports = React.createClass
         </div>
       </section>
       <section className="about-page-content content-container">
-        {if @state.activeTab is 'team'
-          <TeamPage />
-        else
-          <Markdown>{counterpart "about.pageContent.#{@state.activeTab}"}</Markdown>
+        {switch @state.activeTab
```

Comment:
Could we use react-router's RouteHandler component here now instead of tracking activeTab in state?